waku.md

```diff
@@ -62,7 +62,17 @@ Waku was created to incrementally improve in areas that Whisper is lacking in, w
 | **Mail Server** | A node responsible for archiving messages.       |
 | **Mail Client** | A node that requests messages from mail servers. |
 
-## Specification
+## Underlying Transports and Prerequisites
+
+### Use of DevP2P
+
+For nodes to communicate, they MUST implement devp2p and run RLPx. Nodes discover is out of scope for this spec.
```

Comment:
The nodes discovery aspect seems like a mess right now and we need a lot more clarity there... I don't even know where to link to because there seems to be no documentation? Do we even have a namespace in Go that explains whatever we are doing with discoveryv5/rendezvous/staticnodes/etc?

How are things on the Nim side wrt Discovery v4/v5 etc @kdeme?

Reply:
We currently only have discovery v4, so for whisper/waku it is default turned off, as it will have a difficult time finding the whisper nodes of the Status fleet.
Instead a direct connection is made to the fleet.

Discovery v5 is being worked on however.